sdk/python/kfp/compiler/v2_compatible_compiler_test.py

```diff
@@ -13,142 +13,143 @@
 # limitations under the License.
 """Tests for v2-compatible compiled pipelines."""
 
+from kfp.v2.components.types.artifact_types import Artifact
 import os
 import tempfile
 from typing import Callable
 import unittest
 import yaml
 
-from kfp import compiler, components, dsl
-from kfp.components import InputPath, OutputPath
+from kfp import compiler, components
+from kfp import dsl as v1dsl
+from kfp.v2 import dsl
+from kfp.v2.dsl import component, Artifact, InputPath, OutputPath
 
 
+@component
 def preprocess(uri: str, some_int: int, output_parameter_one: OutputPath(int),
                output_dataset_one: OutputPath('Dataset')):
-  '''Dummy Preprocess Step.'''
-  with open(output_dataset_one, 'w') as f:
-    f.write('Output dataset')
-  with open(output_parameter_one, 'w') as f:
-    f.write("{}".format(1234))
+    '''Dummy Preprocess Step.'''
+    with open(output_dataset_one, 'w') as f:
```

Comment:
nit: should we keep the 2-space indentation?

Reply:
Let's switch to 4-space, to be consistent with external Google style guide. WDYT?